src/System.Private.ServiceModel/src/System/ServiceModel/Security/WSSecureConversation.cs

```diff
@@ -0,0 +1,599 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+
+using System;
+using System.Collections.Generic;
+using System.IdentityModel.Selectors;
+using System.IdentityModel.Tokens;
+using System.Runtime;
+using System.ServiceModel;
+using System.ServiceModel.Security.Tokens;
+using System.Xml;
+//using StrEntry = WSSecurityTokenSerializer.StrEntry;
```

Comment:
Removed it -- it was not needed in this file anyway

---
In reply to: [78424958](https://github.com/dotnet/wcf/pull/1515#discussion_r78424958) [](ancestors = 78424958)